server/src/main/java/org/elasticsearch/common/blobstore/EndpointStats.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.common.blobstore;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.xcontent.ToXContentObject;
+import org.elasticsearch.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * The number of calls to an endpoint
+ *
+ * @param operations The number of calls
+ * @param requests The number of calls (including retries)
+ */
+public record EndpointStats(long operations, long requests) implements Writeable, ToXContentObject {
```

Comment:
I'm not sure if "endpoint" is the right word here. I'd be tempted to call it `OperationStats`, but it seems overloaded because `operations` are what we call counts-excluding-retries.